org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.ui/plugin.xml

```diff
@@ -603,8 +603,27 @@
 			type="text">
 		</fileTypes>
 	</extension>
-	<!-- manual additions start here -->
 
+	<!-- code mining -->
+	<extension point="org.eclipse.ui.workbench.texteditor.codeMiningProviders">
+		<codeMiningProvider
+			class="org.eclipse.xtext.example.domainmodel.ui.DomainmodelExecutableExtensionFactory:org.eclipse.jface.text.codemining.ICodeMiningProvider"
+			id="org.eclipse.xtext.example.domainmodel.Domainmodel.CodeMiningProvider"
+			label="Domainmodel Codemining">
+			<enabledWhen>
```

Comment:
@cdietrich What do you think? Should we enable the minings only on the XtextEditor or on all editors that use the same Document type